src/components/Switch/Switch.tsx

```diff
@@ -105,6 +105,10 @@ const Switch = React.forwardRef<HTMLInputElement, SwitchProps>(function Switch(
           as="input"
           cursor="pointer"
           position="absolute"
+          height="100%"
+          width="100%"
+          top={0}
+          left={0}
```

Comment:
@benjamminj after my testing I saw that we also need this

```suggestion
          left={0}
          margin="auto"
```


Without margin:
<img width="164" alt="Screenshot 2022-02-09 at 6 39 42 PM" src="https://user-images.githubusercontent.com/10436045/153247096-572729ce-60f6-4973-9f9b-bf180710ce35.png">

With margin:
<img width="154" alt="Screenshot 2022-02-09 at 6 39 46 PM" src="https://user-images.githubusercontent.com/10436045/153247028-81b33b3a-5401-416c-942c-5ba196c42ae3.png">

Reply:
cool, I'll add that in